server/modules/rendering/html-image-prefetch/renderer.js

```diff
@@ -0,0 +1,23 @@
+const request = require('request-promise')
+
+const prefetch = async (element) => {
```

Comment:
You should handle request failures gracefully. If the server doesn't respond or return an error.

Reply:
fixed